tools/wutool/wutool.cpp

```diff
@@ -92,6 +92,7 @@ void usage(const char * action = nullptr)
                "\n"
                "   archive <workunits> - Archive to xml files [TO=<directory>] [DEL=1] [DELETERESULTS=1] [INCLUDEFILES=1]\n"
                "   restore <filenames> - Restore from xml files [INCLUDEFILES=1]\n"
+               "   importzap <zapreport-filename> <output-helper-directory>\n"
```

Comment:
Not show zapPassword as an option?

Reply:
yeah should do, I've amended it (and usage) - squashed in as trivial change.
@wangkx - please recheck.